packages/location-icon/src/index.tsx

```diff
@@ -60,3 +60,7 @@ export function LocationIcon({
 }
 
 export default LocationIcon;
+
+// Rename styled components for export
+const Styled = S;
+export { Styled };
```

Comment:
It [looks like](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export#syntax) you could condense this into one line like this:

```
export { S as Styled }
```